### PR TITLE
Fixes #58 : Onscreen controller added

### LINKED ISSUE #58: Add buttons to simulate arrow keys in mobile
Let's make it playable at the mobile as well. Add buttons to simulate arrow keys in keyboard so that we can play it on mobile as well.

### PR DESCRIPTION
**Pull request by YashKumarVerma**
This pull request is related to issue #58 

I modified the labyrinth, you can test my changes here:
http://rawgit.com/yashkumarverma/labyrinth/master/index.html

The problem I want to solve is *there are no on screen buttons to play the game*

My solution for the problem is 
*i added the buttons to play the game*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
+    <script type="text/javascript" src="js/levels.js"></script>
+    <script type="text/javascript" src="js/player.js"></script>
+    <script type="text/javascript" src="js/level.js"></script>
     <link rel="stylesheet" href="css/tiles.css">
     <title>Labyrinth</title>
   </head>
   <body>
     <a href="howtoplay.html">How to play?</a>
+    <div>
+
```
Comment (abishekvashok) [inline index.html]: Blank line

File: js/gui.js [hunk 2]
```diff
@@ -47,6 +47,32 @@ var keyHandlers = {
   },
 };
 
+// On Screen Keys
+$("#leftKey").on("click",function(){
```
Comment (niccokunzmann) [inline js/gui.js]: If you put this into the document onload, you can add the scripts to the top.

**Comment by niccokunzmann** [comment]
Cool, thanks!

File: js/gui.js (hunk 2 above)
Comment (YashKumarVerma) [reply js/gui.js]: This thing or the entire script?

Comment (niccokunzmann) [reply js/gui.js]: the entire script. Then, it can be included in the head.

Comment (YashKumarVerma) [reply js/gui.js]: On it

**Pull request merged**